src/canari/maltego/message.py

```diff
@@ -641,6 +642,10 @@ class LinkColor:
     Yellow = '#FFE100'
     Pink = '#FEAFCA'
 
+class LinkDirection:
+    InputToOutput = 'input-to-output'
+    OutputToInput = 'output-to-input'
+
```

Comment:
We should add `Bidirectional = 'bidirectional'` as well as it is also supported by Maltego